test/data/with_main/foo.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * 
```

Comment:
These comments are a bit weird, even if they are in test files.

Reply:
Oops. Added comments.